kubernetes_deploy/development/jobs.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-collect-static
+spec:
+  template:
+    metadata:
+      name: laa-legal-adviser-collect-static
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: app
+        image: "<to be set by deploy_to_kubernetes>"
+        imagePullPolicy: Never
+        command: ["python", "manage.py", "collectstatic", "--noinput"]
+        env:
+          - name: SECRET_KEY
+            value: secret
```

Comment:
I assume this is a quirk in Django and we need `SECRET_KEY` for reasons :tm:

What do you think about changing the value to reflect this? Could we use `required_by_django_but_not_used`, or something similar?

Reply:
Django does need it, but I checked and our app does define a default, so we don't need to pass it from here at all